src/Card/Service.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace PodPoint\Payments\Card;
+
+use PodPoint\Payments\Card;
+use PodPoint\Payments\Customer;
+use PodPoint\Payments\Token;
+
+interface Service
+{
+    /**
+     * Attach a card to a customer.
+     *
+     * @param Token $cardToken
+     * @param Token $customer
+     *
+     * @return Card
+     */
+    public function attach(Token $cardToken, Token $customerToken): Card;
```

Comment:
this feels a bit odd - might need you to walk me through this

Reply:
Yeah I'm not a big fan either.
I'll walk you through the process of linking a card to a customer maybe we can find a cleaner way to do it.